src/strings/csv/en.csv

```diff
@@ -807,7 +807,6 @@ NO_PROJECT,No Project,
 NO_QUANTITY_SET,No quantity set,
 NO_REASSIGNMENTS,No Reassignments,
 NO_REASSIGNMENTS_TEXT,Select a new subzone with quantity in at least one seedlings batch row.,
-NO_SCORE_SELECTED,No score selected,
```

Comment:
May want to leave this in unless you also removed it in Phrase.

Reply:
I can remove it after this is merged in